documentation/Using-Azure-TestFramework.md

```diff
@@ -76,7 +76,7 @@ To use this option, set the following environment variable before starting Visua
 `tenantId`: The [AAD Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
 
 
-	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};ServicePrincipalSecret={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
+	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};Password={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
```

Comment:
@akromm this is deliberately not documented, because this help is for new Framework, Password was legacy support that we do support and will honor but definitely do not want to document it.
Any reason you want to change this?